src/features/BI-2000.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Ontology error message disappears when a valid file is uploaded
+
+    @BI-2000
+    Scenario: Ontology error message disappears when a valid file is uploaded
+        Given a new program is created
+        Given user logs in as "Cucumber Breeder"
+        When user selects "*" on program-selection page
+        And user selects "Ontology" in navigation
+        And user selects "Manage Ontology" button
+        And user selects "Import file" menu item
+        And user uploads Ontology "Bad.xls" file
+		And user selects 'Import' button
+        Then user can see banner contains "Error(s) detected in file, Bad.xls . (See details below.) Import cannot proceed."
+        And user uploads Ontology "test01-ontology.xls" file
+        And user selects 'Import' button
+		When user selects "Confirm" button
+        Then user can see banner contains "Imported ontology terms have been added"
```

Comment:
Given the acceptance criteria includes "THEN the user can see the error message disappear when a valid file is uploaded." should there be a "user can see banner does NOT contain 'THEN the user can see the error message disappear when a valid file is uploaded. '" clause?

Reply:
I have updated to add the "Then user cannot see banner contains "Error(s) detected in file, Bad.xls . (See details below.) Import cannot proceed.".